xclim/indices/_conversion.py

```diff
@@ -1397,7 +1398,8 @@ def potential_evapotranspiration(
 
         lv = 2.5  # MJ/kg
 
-        ra = extraterrestrial_solar_radiation(tasmin.time, lat)
+        time, lat = _chunk_like(tasmin.time, lat, chunks=tasmin.chunks)
```

Comment:
why repeat this here? Are `time` and `lat` modified above? Because python modifies everything all the time by suprise?